src/AirshipBindings.Portable/AirshipBindings.Portable.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+// Information about this assembly is defined by the following attributes. 
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("AirshipBindings.Portable.Abstractions")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("")]
```

Comment:
I think you want to clear out all the empty string ones here and include the shared assembly file